tests/test_evpn_l3_vxlan.py

```diff
@@ -0,0 +1,1726 @@
+        vxlan_obj.check_add_tunnel_nexthop(dvs, ecmp_nhid_list[1], '8.8.8.8', tunnel_name, '00:22:22:22:22:22', '1000')
+        print "\tPass"
+
+        print "\n\nTest VRF IPv6 Route with Tunnel Nexthop update from ECMP to non-ECMP"
+        print "\tTest VRF IPv6 Route with Tunnel Nexthop 8.8.8.8 Update"
+        create_vrf_routes(dvs, "2002::8/64", 'Vrf-RED', '8.8.8.8', "Vlan100", "00:22:22:22:22:22", '1000')
+        vxlan_obj.check_vrf_routes(dvs, "2002::8/64", 'Vrf-RED', '8.8.8.8', tunnel_name, "00:22:22:22:22:22", '1000', 1)
+        print "\tPass"
+
+        print "\tTest Tunnel Nexthop 7.7.7.7 is deleted"
+        vxlan_obj.check_del_tunnel_nexthop(dvs, 'Vrf-RED', '7.7.7.7', tunnel_name, "00:11:11:11:11:11", '1000')
+        print "\tPass"
+
+        print "\tTest Tunnel Nexthop ECMP Group is deleted"
+        vxlan_obj.check_vrf_routes_ecmp_nexthop_grp_del(dvs, 2)
+        print "\tPass"
+
+        print "\tTest VRF IPv6 Route with Tunnel Nexthop 8.8.8.8 Delete"
+        vxlan_obj.fetch_exist_entries(dvs)
+        delete_vrf_routes(dvs, "2002::8/64", 'Vrf-RED')
+
+        vxlan_obj.check_del_tunnel_nexthop(dvs, 'Vrf-RED', '8.8.8.8', tunnel_name, "00:22:22:22:22:22", '1000')
+        vxlan_obj.check_del_vrf_routes(dvs, "2002::8/64", 'Vrf-RED')
+        print "\tPass"
+
+        print "\n\nTest DIP and SIP Tunnel Deletion "
+        print "\tTesting Tunnel Vrf Map Entry removal"
+        remove_vxlan_vrf_tunnel_map(dvs, 'Vrf-RED')
+        vxlan_obj.check_vxlan_tunnel_vrf_map_entry_remove(dvs, tunnel_name, 'Vrf-RED', '1000')
+        print "\tPass"
+
+        print "\tTesting LastVlan removal and DIP tunnel delete for 7.7.7.7"
+        remove_evpn_remote_vni(dvs, 'Vlan100', '7.7.7.7')
+        vxlan_obj.check_vlan_extension_delete(dvs, '100', '7.7.7.7')
+        vxlan_obj.check_vxlan_dip_tunnel_delete(dvs, '7.7.7.7')
+        print "\tPass"
+
+        print "\tTesting LastVlan removal and DIP tunnel delete for 8.8.8.8"
+        remove_evpn_remote_vni(dvs, 'Vlan100', '8.8.8.8')
+        vxlan_obj.check_vlan_extension_delete(dvs, '100', '8.8.8.8')
+        vxlan_obj.check_vxlan_dip_tunnel_delete(dvs, '8.8.8.8')
+        print "\tPass"
+
+        print "\tTesting Vlan 100 interface delete"
+        delete_vlan_interface(dvs, "Vlan100", "2001::8/64")
+        vxlan_obj.check_del_router_interface(dvs, "Vlan100")
+        print "\tPass"
+
+        print "\tTesting Tunnel Map entry removal"
+        remove_vxlan_tunnel_map(dvs, tunnel_name, map_name, '1000', 'Vlan100')
+        vxlan_obj.check_vxlan_tunnel_map_entry_delete(dvs, tunnel_name, vlanlist, vnilist)
+        print "\tPass"
+
+        print "\tTesting SIP Tunnel Deletion"
+        remove_vxlan_tunnel(dvs, tunnel_name)
+        remove_evpn_nvo(dvs, 'nvo1')
+        vxlan_obj.check_vxlan_sip_tunnel_delete(dvs, tunnel_name)
+        vxlan_obj.remove_vrf(dvs, "Vrf-RED")
+        remove_vlan_member(dvs, "100", "Ethernet24")
+        remove_vlan(dvs, "100")
+        print "\tPass\n"
```

Comment:
Please add cases for local static route prefixes for both ipv4 & ipv6, if they are supported.

Reply:
Static route with Tunnel nexthop is currently not supported. And out of scope of this PR.